xblock/fragment.py

```diff
@@ -181,7 +182,7 @@ def add_frags_resources(self, frags):
         for resource in frags:
             self.add_frag_resources(resource)
 
-    def initialize_js(self, js_func, json_args=None):
+    def initialize_js(self, js_func, json_args=None, use_require_js=False):
```

Comment:
I'm still conflicted as to whether this would read more cleanly for the client if it was called ``initialize_amd``. If not, then the first parameter should be renamed from ``js_func`` to make it clearer that it can be either a function or a RequireJS definition that returns a function.